dts/bindings/adc/nxp,kinetis-adc16.yaml

```diff
@@ -20,8 +20,44 @@ properties:
     interrupts:
       required: true
 
+    periodic-trigger:
+      type: boolean
+      required: false
+      description: if periodic trigger enabled
+
     "#io-channel-cells":
       const: 1
 
+    clk-source:
+      type: int
+      required: false
+      description: use alternate clock reference source
```

Comment:
Where are the source values defined (is this from user manual)?

Reply:
yes, it is in the reference manual

Reply:
So this should be a enum.